frontend/playwright/ui/specs/dashboard.spec.js

```diff
@@ -52,3 +52,45 @@ test("Lists files in the drafts page", async ({ page }) => {
     dashboardPage.page.getByRole("button", { name: /New File 2/ }),
   ).toBeVisible();
 });
+
+test("User hasn't an empty placeholder", async ({ page }) => {
+  const dashboardPage = new DashboardPage(page);
+  await dashboardPage.goToDashboard();
+  await expect(
+    dashboardPage.page.getByTestId("empty-placeholder"),
+  ).toBeHidden();
+});
+
+test("User has context menu options for edit file", async ({ page }) => {
+  await DashboardPage.mockRPC(
+    page,
+    "get-all-projects",
+    "dashboard/get-all-projects.json",
+  );
+
+  const dashboardPage = new DashboardPage(page);
+  await dashboardPage.setupDrafts();
+  await dashboardPage.goToDrafts();
+
+  const button = dashboardPage.page.getByRole("button", { name: /New File 2/ });
+  await button.click();
+  await button.click({ button: "right" });
+
+  await expect(dashboardPage.page.getByTestId("file-rename")).toBeVisible();
+  await expect(dashboardPage.page.getByTestId("file-duplicate")).toBeVisible();
+  await expect(dashboardPage.page.getByTestId("file-add-shared")).toBeVisible();
+  await expect(dashboardPage.page.getByTestId("file-delete")).toBeVisible();
+});
+
+test("User has create file button", async ({ page }) => {
+  const dashboardPage = new DashboardPage(page);
+  await dashboardPage.setupDrafts();
+  await dashboardPage.goToDrafts();
+  await expect(dashboardPage.page.getByTestId("new-file")).toBeVisible();
+});
+
+test("User has add font button", async ({ page }) => {
+  const dashboardPage = new DashboardPage(page);
+  await dashboardPage.goToFonts();
+  await expect(dashboardPage.page.getByText("ADD CUSTOM FONT")).toBeVisible();
```

Comment:
By default, `getByText` is case insensitive, so no need to go all caps here. Also, implementation-wise, it seems this text should be written as "Add custom font" and use CSS to display it with uppercase characters.